### PR TITLE
delete caledar

### PR DESCRIPTION
**Pull request by phornsopheak**

### CODE REVIEW

File: app/controllers/calendars_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      flash[:success] = t "calendars.deleted"
+    else
+      flash[:danger] = t "calendars.not_deleted"
+    end
+    redirect_to "#"
+  end
+
+  private
+  def load_calendar
+    @calendar = Calendar.find params[:id]
```
Comment (tqtrung09) [inline app/controllers/calendars_controller.rb]: use find_by

**Comment by nhunght** [comment]
ok

**Comment by nguyenthaison** [comment]
oke

**Comment by tqtrung09** [comment]
Ok

File: app/controllers/calendars_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CalendarsController < ApplicationController
+  load_and_authorize_resource
+
+  def destroy
+    if @calendar.destroy
+      flash[:success] = t "calendars.deleted"
+    else
+      flash[:danger] = t "calendars.not_deleted"
+    end
+    redirect_to "#"
```
Comment (dieunb) [inline app/controllers/calendars_controller.rb]: ???

**Pull request merged**